locations/spiders/itsu_gb.py

```
@@ -8,12 +8,14 @@ class ItsuGBSpider(SitemapSpider):
     name = "itsu_gb"
     item_attributes = {"brand": "itsu", "brand_wikidata": "Q6094914"}
     sitemap_urls = ["https://www.itsu.com/sitemap_pages.xml"]
-    sitemap_rules = [("/location/", "parse")]
+    sitemap_rules = [(r"/location/[^/]+/[^/]+/", "parse")]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)